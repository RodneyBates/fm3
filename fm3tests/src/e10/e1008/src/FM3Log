Start unit Main.m3
[32mFM3: [0mMain.m3 Compiling /home/rodney/proj/m3/fm3/git/fm3tests/src/e10/e1008/src/Main.m3...
Start unit Main.i3
[32mFM3: [0mMain.i3 Compiling /home/rodney/proj/m3/fm3/git/compiler/AMD64_LINUX/../lib/Main.i3...
[32mFM3: [0mMain.i3 Pass 1 output file Main.i3.FM3Pass1 has 70 bytes.
[32mFM3: [0mMain.i3 Patch stack Main.i3.FM3Patch peak size = 35 bytes.
[32mFM3: [0mMain.i3 Pass 2 output file Main.i3.FM3Pass2 has 62 bytes.
[32mFM3: [0mMain.i3 Finished compiling Main.i3.
End unit Main.i3
[32mFM3: [0mMain.m3 Pass 1 output file Main.m3.FM3Pass1 has 57 bytes.
[32mFM3: [0mMain.m3 Patch stack Main.m3.FM3Patch peak size = 25 bytes.
[32mFM3: [0mMain.m3 Pass 2 output file Main.m3.FM3Pass2 has 55 bytes.
[32mFM3: [0mMain.m3 Finished compiling Main.m3.
End unit Main.m3
