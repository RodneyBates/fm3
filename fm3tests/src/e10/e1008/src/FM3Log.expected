[32mFM3:   [0mCompiling with command line: 
         /home/rodney/proj/m3/fm3/git/compiler/AMD64_LINUX/FM3 --disasm --no-disasm-verbose --exprs --no-expr-addrs --no-std-sources Main.m3"
[32mFM3:   [0mGetting dependencies of 
         /home/rodney/proj/m3/fm3/git/fm3tests/src/e10/e1008/src/Main.m3 ...
[32mFM3:   [0mCompiling Main.m3 ...
[32mFM3:   [0mFinished compiling Main.m3.
